pkg/operator2/idp.go

```diff
@@ -343,3 +344,12 @@ func encodeOrDie(obj runtime.Object) []byte {
 	}
 	return bytes
 }
+
+func transportForTrustedCA(serverName string, certData []byte, keyData []byte) (http.RoundTripper, error) {
+	caData, err := ioutil.ReadFile(operatorTrustedCAFile)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read %s: %v", operatorTrustedCAFile, err)
```

Comment:
Why are failing on an optional CA?

Reply:
Yeah, that's stupid, I've gone a bit different path in latest update